file-formats/chkv/type/zif_aff_chkv_v1.intf.json

```diff
@@ -1,7 +1,8 @@
 {
-  "$schema": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/chkv/chkv.json",
+  "$schema": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/intf/intf.json",
```

Comment:
this looks wrong

Reply:
yes, this is a mistake.Thank you for finding it. I will correct it

Reply:
"$schema": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/intf/intf.json"
is correct. This is a serialized interface and therefore intf/intf is correct.
It is the interface which contains the abap type of chkv objects.

I also always stumble over this. It's annoying :-)